test/measures/measures-data-spec.js

```diff
@@ -6,156 +6,159 @@ const path = require('path');
 const parse = require('csv-parse/lib/sync');
 
 const mipsDataFormat = require('../../index.js');
-const measuresData = mipsDataFormat.getMeasuresData();
 const actualAciRelation = require('../../util/measures/aci-measure-relations.json');
 const actualCpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
 const actualMeasureSpecificationData = parse(fs.readFileSync(path.join(__dirname, '../../util/measures/measurePDF-Specification.csv'), 'utf8'));
 
-describe('measures data json', function() {
-  const measureIds = _.map(measuresData, 'measureId');
+const performanceYears = [2017, 2018];
+for (const year of performanceYears) {
+  const measuresData = mipsDataFormat.getMeasuresData(year);
+  describe(year + ' measures data json', function() {
+    const measureIds = _.map(measuresData, 'measureId');
```

Comment:
This is a poor diff... everything below this line through the end of the file is unchanged other than being indented by two spaces since it's now inside a for..of loop.

Reply:
I wonder if we'll want to create separate spec files for 2017 and 2018. 2018 for example, will have all ACI measures renamed so those tests will need to be rewritten.

Reply:
Makes me wonder if we should have different schemas too - there would be a lot of duplication but it would be easier to validate something like oh 2017 has measures that start with ACI and 2018 has measures that start with PI.

Reply:
Yeah I think that's a good idea! Will do.

Reply:
OK I created the separate spec files but separate schemas is a lot more involved so I'll leave that for the next PR